lib/fishjam/rpc_client.ex

```diff
@@ -30,9 +37,16 @@ defmodule Fishjam.RPCClient do
   """
   @spec multicall(module(), atom(), term(), timeout()) :: list(term)
   def multicall(module, function, args \\ [], timeout \\ :infinity) do
+    start_time = System.monotonic_time(:millisecond)
+
     nodes()
     |> :erpc.multicall(module, function, args, timeout)
     |> handle_result()
+    |> then(fn result ->
+      emit_rpc_duration_event([:fishjam, :rcp_client, :multicall], start_time)
+
+      result
+    end)
```

Comment:
I think we could use [tap function](https://hexdocs.pm/elixir/1.17.0/Kernel.html#tap/2) instead.
```suggestion
    |> tap(&emit_rpc_duration_event([:fishjam, :rcp_client, :multicall], start_time))
```
```suggestion
    |> then(fn result ->
      emit_rpc_duration_event([:fishjam, :rcp_client, :multicall], start_time)

      result
    end)
```